src/common/SPELLS/demonhunter.ts

```diff
@@ -154,21 +154,16 @@ const spells: SpellList<DemonHunterSpell> = {
     painCost: 30,
   },
   IMMOLATION_AURA: {
-    id: 178740,
-    firstStrikeSpellId: 187727,
+    id: 258920,
+    firstStrikeSpellId: 187727, // fix it
```

Comment:
I have no idea. I just added the comment with this because, well.. the game shows a completely different ID to the old one so I just added it there..

Reply:
I'm just saying that we can remove `firstStrikeSpellId` here and from `DemonHunterSpell` altogether since its unused.
There is a merge conflict at this line anyways, so while you're at it please remove this remnant completely.